Team417_PowerPlay/src/main/java/org/firstinspires/ftc/team417_PowerPlay/BaseOpMode.java

```diff
@@ -42,4 +57,21 @@ public void initializeHardware() {
         motorBL.setPower(0);
         motorBR.setPower(0);
     }
+
+    public void mecanumDrive(double Xdrive, double Ydrive, double Turningdrive) {
+        double FLPower;
+        double FRPower;
+        double BLPower;
+        double BRPower;
```

Comment:
Please use camelCase for your variable names to be consistent with everything else. Also powerFL is more consistent with motorFL.
```suggestion
    public void mecanumDrive(double xDrive, double yDrive, double turningDrive) {
        double powerFL;
        double powerFR;
        double powerBL;
        double powerBR;
```